openghg/retrieve/_access.py

```diff
@@ -544,8 +544,10 @@ def get_bc(
 
 
 def get_footprint(
-    site: str,
     domain: str,
+    obs_region: str | None = None,
```

Comment:
Could we move `obs_region` to be below `site` and `satellite` (as we have for the `standardise_footprint` function)? I think keeping this consistent would emphasise the relative importance of the keywords as we're still going be including `site` footprints most often.